docs-new/docs/components/lifecycle-operator/deployment-flow.md

```diff
@@ -165,22 +168,27 @@ Additional phases/states exist,
 such as those that describe what happens when something fails.
 
 Whenever something in the system happens (we create a new resource, etc.)
```

Comment:
Does an action usually describe what happens when something fails or does it take an action that is appropriate to deal with the failure?

Reply:
events show us actions that already happened (evaluation failed/succeeded etc.)

Reply:
Maybe we could rephrase this section by saying that during the lifetime of the controllers, some resources are created or deleted to pass information, and each change in a resource also causes a new reconciliation loop in the controller. The events linked to this phase are:

Reply:
Adapted